server/src/main/java/org/elasticsearch/monitor/jvm/HotThreads.java

```diff
@@ -104,6 +105,34 @@ public static void logLocalHotThreads(Logger logger, Level level, String prefix,
         }
     }
 
+    /**
+     * Capture and log the current threads on the local node. Unlike hot threads this does nto sample and captures current state only.
+     * Useful for capturing stack traces for unexpectedly-slow operations in production. The resulting message might be large so it is
+     * split per thread and logged as multiple entries.
+     *
+     * @param logger        The logger to use for the logging
+     * @param level         The log level to use for the logging.
+     * @param prefix        The prefix to emit on each chunk of the logging.
+     * @param referenceDocs A link to the docs describing how to decode the logging.
+     */
+    public static void logLocalCurrentThreads(Logger logger, Level level, String prefix, ReferenceDocs referenceDocs) {
+        if (logger.isEnabled(level) == false) {
+            return;
+        }
+
+        try (var writer = new StringWriter()) {
+            new HotThreads().busiestThreads(500).threadElementsSnapshotCount(1).ignoreIdleThreads(false).detect(writer, () -> {
```

Comment:
I wonder whether we should `ignoreIdleThreads(true)`? Including them provides a complete picture but I often found skipping them (like 90% of the output) is quite painful. Since this new method is meant for mostly internal investigation and per thread, I wonder whether we could do away with idle threads.

Reply:
Good point. This was copied from 
https://github.com/elastic/elasticsearch/blob/7344b151fb323a4f106fe5a5819b49f70fc35278/server/src/main/java/org/elasticsearch/monitor/jvm/HotThreads.java#L102
but I also do not find them very informative.

@DaveCTurner was there a reason to enable them explicitly?

Reply:
We can drop them here, sure.